### DISCUSSION TITLE
Using the element inspector

### DISCUSSION THREAD

**Discussion by tanrax** [ionic-team/capacitor#3322] [Q&A] [answered]
I'm trying to inspect the HTML in device with chrome://inspect/#devices or Safari tools, but they don't connect. So I read in the documentation that it's only possible using the ionic client. But... what do I do if it's not an ionic project?
Thanks in advance

**Comment by jcesarmobile** [comment, marked as answer]
Capacitor apps are inspectable without doing anything on the app and no matter what framework you use. It's not a feature provided by Capacitor, but by the WebView. The only thing Capacitor does is to enable this feature by default in Android debug builds (on iOS nothing needs to be done)

On Android the device needs to have developer options enabled and USB debugging.
On iOS in Settings App go to Safari > Advanced and toggle on Web Inspector.

If you say "don't connect", I guess you see the devices but fail to inspect, so you probably already did what I mentioned. Then try restarting the devices, fully closing safari/chrome and reopen, or even restart the computer.

**Reply by tanrax** [reply]
Thanks!
